autoload/internal_buffer.vim

```diff
@@ -650,17 +651,23 @@ fu! s:InternalBuffer.RenderUi() dict abort
     call self.RenderUiUsagesList(self.usages_grep_results, self.len() + 1)
   endif
 
-  call self.AddLine([ self.CreateItem("help_link", "> Help", g:AnyJumpGetColor('heading_text')) ])
+  call self.ShowHelpSection()
```

Comment:
Can you rename this method to "HelpSection"? Since showing or not showing is now optional. And after this i think this commit can be merged. Thanks!

Reply:
Thank you, updated it.